assemble-and-restart

```diff
@@ -13,6 +13,23 @@ else
     /usr/libexec/s2i/assemble
 fi
 
+### 
+# Check "ODO_S2I_DEPLOYMENT_DIR" environment variable and if it's present,
+# copy content of directory to /opt/app-root/backup directory
+# Ref: https://github.com/redhat-developer/odo/issues/445
+
+if [ ! -d /opt/app-root/backup ]; then
+    mkdir  /opt/app-root/backup
```

Comment:
why not just use `mkdir -p`?
Shouldn't we move this to `if [ -z "$ODO_S2I_DEPLOYMENT_DIR" ]` `else` block as we don't need to create this if the variable is not set